vscode-trace-webviews/src/trace-explorer/opened-traces/vscode-trace-explorer-opened-traces-widget.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import { ReactOpenTracesWidget } from '@trace-viewer/react-components/lib/trace-explorer/trace-explorer-opened-traces-widget';
+import { VsCodeMessageManager } from '../../common/vscode-message-manager';
+import { Menu, Item, useContextMenu, ItemParams } from "react-contexify";
+import { TspClientProvider } from '../../common/tsp-client-provider-impl';
+import { ITspClientProvider } from '@trace-viewer/base/lib/tsp-client-provider';
+import { Experiment } from 'tsp-typescript-client/lib/models/experiment';
+import { signalManager, Signals } from '@trace-viewer/base/lib/signals/signal-manager';
+import '../../style/trace-explorer.css';
+import '@trace-viewer/react-components/style/trace-explorer.css';
+import '../../style/react-contextify.css';
+import { ExperimentManager } from '@trace-viewer/base/lib/experiment-manager';
+
+interface OpenedTracesAppState {
+  tspClientProvider: ITspClientProvider | undefined;
+}
+
+const MENU_ID = 'traceExplorer.openedTraces.menuId';
+
+class TraceExplorerOpenedTraces extends React.Component<{}, OpenedTracesAppState>  {
```

Comment:
Do we still call the classes TraceExplorer even if the view is now called Trace Viewer?

Reply:
Good point. I'm aware of it. In the theia-trace-extension we have the same mismatch. The user facing strings were changed from "Trace Explorer" to "Trace Viewer", but the classes and variables stayed the same. To not introduce more confusion, I kept the same discrimination in the vscode-trace-extension than in the theia-trace-extension. But yeah, a code clean-up should be done in both repos.

IMHO, the discrimination between "Trace Explorer" for traces/analysis and "Trace Viewer" for the widgets part made sense, but the majority where against it.